package.json

```diff
@@ -30,8 +30,8 @@
   },
   "scripts": {
     "coverage": "nyc --reporter=lcov --reporter=text npm test",
-    "test": "mocha --exit -R spec --timeout 5000 test/*.js",
-    "lint": "eslint lib/**.js test/**.js",
+    "test": "NODE_ENV=test mocha --exit -R spec --timeout 5000 test/*.js",
+    "lint": "eslint lib/**/*.js test/**/*.js",
```

Comment:
This makes linter to run for all files in all dirs

Reply:
I thought `npm test` implied setting `NODE_ENV=test`...

Reply:
Sorry for misleading comment, I had to go different path: https://github.com/brightcove/hot-shots/pull/130/files#diff-831d41ad895e89c8dffa55c52a8bd1bcR112